README.md

```diff
@@ -191,14 +198,20 @@ _You are advised to read this section before using django-components in producti
 
```

Comment:
In v1, this section on static files could be removed, but until then it makes sense to keep it